### ISSUE TITLE
[BUG] Can't view logs in webUI

### ISSUE THREAD

**Issue by bpatel928** [bitfocus/companion#2425]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I am getting this error while trying to see the Logs in the 2.4.2 web ui of companion. 

Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Berrno%2C%20code%2C%20syscall%2C%20address%2C%20port%7D for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

I am not sure what I need to go and look at in the install directory to fix this issue. 

The build that I see that I am using is 

2.4.2 (2.4.2+4911-unknown-fcb5a863)

Ideally I would like to fix the issue without having to reinstall companion entirely. 

### Steps To Reproduce

Go to web ui of comapanion and go to logs 

### Expected Behavior

if the same Issue I have is there, then it'll fail to open. 

### Environment (please complete the following information)

```markdown
- OS:raspbian buster
- Browser:Chrome
- Companion Version: 2.4.2
```

**Comment by bryce-seifert** [comment]
I noticed this in 2.4.2, and my best guess is it was the VICREO module causing this error in my case. Are you using that module?

**Comment by bpatel928** [comment]
Yes, I am using the vicreo module.

**Comment by bryce-seifert** [comment]
It might be worth trying a v3 Beta to see if it persists, I haven’t seen the issue there but I also haven’t been testing Vicreo extensively on that version

**Comment by Julusian** [comment]
I am pretty sure this is fixed already in 3.0, if not this can be reopened
